### ISSUE TITLE
REVIEW: harmonize web tutorial with JOSS manuscript examples

### ISSUE THREAD

**Issue by earth-chris** [jbferet/prospect#10]
(Part of JOSS review https://github.com/openjournals/joss-reviews/issues/6027)

Both the [web documentation](https://jbferet.gitlab.io/prospect/articles/prospect1.html) and the JOSS manuscript include examples for running PROSPECT under a number of scenarios. The web documentation appears to be less recent than the manuscript.

As it stands, I cannot execute the first line in the tutorial:

```
library(prospect)
LRT_default <- PROSPECT(SpecPROSPECT)
```

It looks like this throws an error that checks for whether to run PROSPECT-D or PROSPECT-PRO:

```
Error in if (!LMA == 0 & (PROT > 0 | CBC > 0)) {: argument is of length zero
Traceback:

1. PROSPECT(SpecPROSPECT)
```

This example should be updated so it executes properly. Other examples that specify the minimum default arguments succeed, but this documented snippet doesn't.

There are many additional examples provided in the JOSS manuscript that users would benefit from having in the web docs. As a user, I am most interested in the inversion methods, and example inversion code is included in the manuscript but not in the web docs. I recommend updating the web docs to include this deeper set of code examples, as it is difficult (at the moment, impossible) to copy code snippets from the manuscript and run them to verify execution.

**Comment by jbferet** [comment]
@earth-chris thank you for raising this issue. This should now be fixed in prospect 1.5.0. 
The code and web documentation are now up-to-date. Please tell me if any error remains.

**Comment by earth-chris** [comment]
thanks for the fix! i can execute the example code with no issues.
